LF351 symbol added

johnfwhitmore opened the pull request:
Re-added LF351 with correct version of Kicad

Datasheey - https://www.st.com/resource/en/datasheet/lf351.pdf

![Screenshot from 2020-03-17 20-26-52](https://user-images.githubusercontent.com/15182161/76898565-dc10fe00-688d-11ea-8c26-01fd1169537f.png)

cpresser commented on Amplifier_Operational.dcm:
```diff
@@ -180,6 +180,18 @@ K single jfet opamp
 #
+$CMP LF351D
+D Wide bandwidth single JFET operational amplifier, SOIC-8
+K Singel, OpAmp, JFET
```
typo. 'singel' vs. 'single'
Also commas don't belong into keywords.
But description should have commas.

cpresser commented on Amplifier_Operational.dcm:
```diff
@@ -180,6 +180,18 @@ K single jfet opamp
+$ENDCMP
+#
+$CMP LF351N
+D Wide bandwidth single JFET operational amplifier, DIP8
```
DIP-8 instead of DIP8 please.

cpresser commented on Amplifier_Operational.dcm:
```diff
@@ -180,6 +180,18 @@ K single jfet opamp
+#
+$CMP LF351N
+D Wide bandwidth single JFET operational amplifier, DIP8
+K Single, OpAmp, JFET
```
same as above, no commas here.

johnfwhitmore commented:
Sorry meant to do that yesterday, but got distracted.
And sorry about the mistake in the first place. I got confused with the last PR, which I deleted. The last comment said something about commas and I added them to keyword field not remembering the exact request that was made. Having deleted the PR and the branch I couldn't get back to the exact comment.

cpresser commented:
There is no need to rush. It is nice to finish a PR fast, but most take weeks or even months.
Can you please fix the last thing from my review: `DIP-8 instead of DIP8 please`
Then this is ready to merge.
